### DISCUSSION TITLE
`useAuthEmulator` documentation and API

### DISCUSSION THREAD

**Discussion by akauppi** [firebase/firebase-js-sdk#4526] [Q&A]
First, `useAuthEmulator` should likely be documented [here](https://modularfirebase.web.app/reference/auth/). It's not.

Second, would this be a good chance to unify the `use...Emulator` calls to follow the same pattern.

```
useFirestoreEmulator( firestore, host, port );
useFunctionsEmulator( functions, host, port );
```

..but 

```
useAuthEmulator(auth, url, options?: {
  disableWarnings: boolean;
}
```

[1](https://modularfirebase.web.app/reference/firestore_.usefirestoreemulator) [2](https://modularfirebase.web.app/reference/functions.usefunctionsemulator)

**Comment by davideast** [comment]
Hey, @akauppi! That API reference is generated, so there's probably something a miss. Thanks for letting us know, we'll get it updated.

[We did though just put up a "Common Use Case" for it yesterday.](https://modularfirebase.web.app/common-use-cases/authentication/#connect-to-emulator) So I hope that helps for the time being.

**Reply by akauppi** [reply]
@davideast I took away the "accepted answer" since the page is still missing the entry. Let me know when I can expect to test, again.
